thunder/tests/opinfos.py

```diff
@@ -5005,9 +5005,18 @@ def make_with_extremal_value(shape, extremal, percentage=0.5):
     # `torch` eager - gradients max(dim) propagates gradient only to a single index in the source tensor
     # `thunder` - gradients are distributed evenly.
     # So, we use the function below to create tensor with unique values.
-    def make_unique_t(shape):
-        # Add two random inputs, so it is unlikely to have same value across the tensor elements.
-        return make(shape) + make(shape)
+    def make_t(shape):
+        if dtype.is_floating_point and requires_grad:
+            # Use `linspace` to create tensor with unique values.
+            numel = math.prod(shape)
```

Comment:
Let's add a comment explaining that we do that so that finite diff is happy

Reply:
Done :)